libexec/decomposer/decomposer-install

```diff
@@ -94,18 +94,25 @@ install_library() {
   revision_alt_type=$( git cat-file -t "v${revision}" 2> /dev/null )
   revision_reset=
 
-  if git branch -a | grep -q "/origin/${revision}"; then
-    revision_reset="origin/${revision}"
+  # use the origin for master as the default origin
+  default_remote="$(git config --get branch.master.remote)"
+  if git remote get-url "$default_remote" | grep -q "${url}"; then
+    printf -v "${status_text_variable}" 'remote URL has changed'
+    git remote set-url "$default_remote" "${url}"
+  fi
```

Comment:
Use `git config --get remote.origin.url` instead of the `git remote get-url` call. `git remote get-url` returns the unaliased URL. If you use an aliased URL in the config this will always mismatch. `git config` returns the aliased URL though, so this should match.